README.rst

```diff
@@ -2,51 +2,122 @@
-
-* {{ cookiecutter.hosting }} repo: {{ cookiecutter.hosting_repo_url }}
-* Free software: MIT license
-
-Features
---------
-Fully working scaffold with the following out-of-the-box and running features (although
-external environments like GitLab-CI_, PyPI_ and `Read the Docs`_ require to be set up
-separately):
-
-* Pytest_: Unit tests
-* Behave_: BDD/Functional tests
-* Flake8_: Linting/Style guide enforcement
-* Black_ & isort_: Uncompromising formatting and automated import sorting
-* Mypy_: Static typing
-* Coverage_: Code coverage
-* Poetry_: Easy packaging and dependency management, set up to isolate the different
-  test dependencies (unit tests, bdd, ...) for speedy test runs
-* Pyenv_: Simple Python version management to install multiple versions on your host
-* Make_: Used as single doorway to all your tasks:
-
-  * For setup (development environment and machine, CI/CD pipeline job environments)
-  * For test tasks
-  * For the whole development and release lifecycle
-
-* Tox_ testing: Set up to easily test for Python 3.6, 3.7, 3.8 and used as single source
-  for all your test invocations
-* Sphinx_ docs: Documentation ready for generation with, for example, `Read the Docs`_
-* GitLab-CI_: CI-CD pipeline with:
-
-  * Test phase running all tests in parallel jobs
-  * Release phase (see semantic-release below)
-
-* Semantic-release_: Pre-configured version bumping, changelog generation, auto-release
-  to PyPI_ and GitLab (yes... not a Python written tool... but works *really* well and
-  has way more advanced features than any of the existing Python tools)
-* Command line interface (optional) using Cleo_ (default), Click_ or Argparse_
+.. image:: https://github.com/opinionated-digital-center/python-library-project-generator/workflows/Test%20and%20make%20release/badge.svg
+    :target: https://github.com/opinionated-digital-center/python-library-project-generator/actions?query=workflow%3A%22Test+and+make+release%22
+
+.. image:: https://github.com/opinionated-digital-center/python-library-project-generator/workflows/Test%20GitLab%20Pipeline/badge.svg
+    :target: https://github.com/opinionated-digital-center/python-library-project-generator/actions?query=workflow%3A%22Test+GitLab+Pipeline%22
+
+* GitHub repo: https://github.com/opinionated-digital-center/python-library-project-generator .
+* Free software: MIT license.
+
+The "Opinionated Python Library Project Template" is a fully tested, feature rich
+`Cookiecutter`_ scaffold to kick start your Python library and command line interface
+development.
+
+It is "opinionated" as it reflects strong structural and tooling choices,
+which we believe help developers being more effective within Digital Centers. It can
+however seamlessly be used for Open Source development (works also out-of-the-box with
+gitlab.com, github.com, pypi.org and readthedocs.org, as a standalone project).
+
+In addition to the intrinsic features offered by the template (see "Features list"
```

Comment:
Use anchor and link for `features list` ?

Reply:
I can't make it work on github... `rst` is not as well supported as `md`... we might want to switch in the future for documents on the `root` directory.